manifests/mod/dav_svn.pp

```diff
@@ -4,7 +4,7 @@
   Class['::apache::mod::dav'] -> Class['::apache::mod::dav_svn']
   include ::apache
   include ::apache::mod::dav
-  if($::operatingsystem == 'SLES' and $::operatingsystemmajrelease < '12'){
+  unless stdlib::os_version_gte('SLES', '12') {
```

Comment:
I didn't namespace the function so it should be on the top level. Perhaps it should be?

Reply:
Unfortunately the method seems to be causing error's on the unit test's :(
```
     Failure/Error: it { is_expected.to contain_class('apache::params') }

     Puppet::PreformattedError:
       Evaluation Error: Unknown function: 'stdlib::os_version_gte'. (file: /Users/david.swan/GitHub/puppetlabs-apache/spec/fixtures/modules/apache/manifests/mod/fcgid.pp, line: 5, column: 6) on node david.swan-c02l57z6fft1
```
Might have to remove it.

Reply:
If you call `os_version_gte` (without `stdlib::`) I'd expect it works.

Reply:
Ok, it's throwing an unknown function error when the acceptance test's are run, tried a few things but nothing seemed to get rid of it so I just ripped it out.

Reply:
That's probably because the acceptance test installs a released version. Since the function is not yet in a released version, it's a problem. It'd also be needed to raise the minimum version in metadata.json.

Reply:
Yeh, that sounds right.
Apologies, but at this point I think I'm just gonna go ahead without it since it only resolved down to a single use in the end.

Reply:
Oh yes, that totally makes sense. I often also don't want to depend on the very latest version of a very common module (which stdlib absolutely is) to give users a chance to upgrade. So I'm fine with leaving the cleanup to a future PR.

Reply:
IS everything else good with my pr?? Any more changes you think are needed/beneficial???